sync/sync_head.go

```diff
@@ -186,7 +193,68 @@ func (s *Syncer[H]) verify(ctx context.Context, newHead H) (bool, error) {
 			"reason", verErr.Reason)
 	}
 
-	return verErr.SoftFailure, err
+	return false, err
+}
+
+// verifySkipping will try to find such headers in range (subjHead, networkHeader)
```

Comment:
If someone have a better doc comment: feel free to share it.

Reply:
```
verifySkipping performs a bifurcated header verification process such that it tries to find a header (or several headers if necessary) between the networkHead and the subjectiveHead such that non-adjacent (or in the worst case adjacent) verification passes and the networkHead can be verified as a valid sync target against the syncer's subjectiveHead.
```

Reply:
Updated, thank you.